pkg/controllers/dashboard/fleetcharts/controller.go

```diff
@@ -69,6 +70,12 @@ func (h *handler) onSetting(key string, setting *v3.Setting) (*v3.Setting, error
 		"global":       systemGlobalRegistry,
 	}
 
+	if !features.Gitops.Enabled() {
+		fleetChartValues["gitops"] = map[string]interface{}{
+			"enabled": false,
+		}
```

Comment:
Are we using a map in anticipation of future values being added?

Reply:
actually I can change it to 
```
fleetChartValues["gitops"] = map[string]interface{}{			
  "enabled": features.Gitops.Enabled(),		
}
```